src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Diagnostics/RazorDiagnosticsPublisher.cs

```diff
@@ -151,61 +151,61 @@ internal void ClearClosedDocuments()
         try
         {
             lock (PublishedRazorDiagnostics)
-            lock (PublishedCSharpDiagnostics)
-            {
-                ClearClosedDocumentsPublishedDiagnostics(PublishedRazorDiagnostics);
-                ClearClosedDocumentsPublishedDiagnostics(PublishedCSharpDiagnostics);
+                lock (PublishedCSharpDiagnostics)
```

Comment:
Having nested locks at the same indent level with just one set of braces is pretty standard. However, if you'd rather indent the nested lock statement, please add braces for readability.

Reply:
This is 100% not intentional, I just have a habit of pressing Ctrl+E, D a fair amount. Will fix up, thanks for catching it.